.github/PULL_REQUEST_TEMPLATE_nolabels

```diff
@@ -0,0 +1,13 @@
+## ATTENTION - Release Note ack required!
```

Comment:
@bgrant0607, I've updated the file in this PR and the above template and have added a placeholder for a docs checkbox based on our conversation today.  

I'm not sure if we want this in here yet or at all or in the current form, but I wanted to put the placeholder here at least.  Having it in the template for now at least raises the visibility of companion doc changes.  We can add a munger maybe later to actually ensure it's ticked.

cc @janetkuo @pwittrock